src/test/java/lotto/domain/prize/WinningPrizeTest.java

```diff
@@ -23,4 +22,21 @@ public class WinningPrizeTest {
         assertThat(prize.winAmount(LottoPrize.THIRD.getMatch())).isEqualTo(2);
         assertThat(prize.winAmount(LottoPrize.FOURTH.getMatch())).isEqualTo(2);
     }
+
+    @Test
+    void 입력받은_로또_당첨_번호로_상금_총액을_정상_계산() {
+        WinningPrize prize = new WinningPrize();
+
+        prize.record(LottoPrize.FIRST.getMatch());
+        prize.record(LottoPrize.SECOND.getMatch());
+        prize.record(LottoPrize.THIRD.getMatch());
+        prize.record(LottoPrize.THIRD.getMatch());
+        prize.record(LottoPrize.FOURTH.getMatch());
+        prize.record(LottoPrize.FOURTH.getMatch());
+
+        int prizeTotal = LottoPrize.FIRST.prize(1) + LottoPrize.SECOND.prize(1)
+                +LottoPrize.THIRD.prize(2) + LottoPrize.FOURTH.prize(2);
+
+        assertThat(prize.totalPrize()).isEqualTo(prizeTotal);
```

Comment:
기대 값은 하드코딩하는 것이 의도를 쉽고 빠르게 파악 가능한 것 같아요 
```suggestion
        int expected = 2_001_610_000;

        assertThat(prize.totalPrize()).isEqualTo(expected);
```